frontend/app/src/components/settings/accounting/rule/AccountingRuleWithLinkedSettingDisplay.vue

```diff
@@ -42,26 +42,28 @@ const value: ComputedRef<boolean> = computed(() => {
 </script>
 
 <template>
-  <RuiBadge
-    placement="top"
-    size="sm"
-    color="secondary"
-    class="[&_span]:!px-0"
-    :value="!!selectedLinkableSetting"
-  >
-    <template #icon>
-      <RuiTooltip
-        v-if="selectedLinkableSetting"
-        :popper="{ placement: 'top' }"
-        open-delay="400"
-      >
-        <template #activator>
-          <RuiIcon size="12" name="links-line" />
-        </template>
-        <div>{{ t('accounting_settings.rule.value_overwritten') }}</div>
-        <div class="font-bold">{{ selectedLinkableSetting.label }}</div>
-      </RuiTooltip>
-    </template>
-    <SuccessDisplay size="28" :success="value" />
-  </RuiBadge>
+  <div class="flex items-center justify-center">
```

Comment:
what was the problem with the previous approach without the wrapper div?

When I checked it I didn't see anything being off